api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java

```diff
@@ -193,6 +192,37 @@ public Order getRevisionOrder(Order order) throws APIException {
 	}
 	
 	/**
+	 * @since 1.12
```

Comment:
The DAO is not public to API consumers, so you normally don't need @since in the DAO